src/plugin/configuration.rs

```diff
@@ -3,6 +3,9 @@ use bevy::prelude::{FromWorld, Resource, World};
 use crate::math::{Real, Vect};
 use crate::plugin::RapierContext;
 
+#[cfg(doc)]
+use crate::prelude::TransformInterpolation;
+
 #[cfg(doc)]
 use rapier::dynamics::IntegrationParameters;
```

Comment:
Can be combined under a single cfg:
```suggestion
#[cfg(doc)]
use {
    crate::prelude::TransformInterpolation,
    rapier::dynamics::IntegrationParameters,
};
```